Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py

```diff
@@ -96,7 +96,10 @@ def build_indicators(self, args: Dict[str, Any], data: list):
                 if not data:
                     data = self.get_recursively(eachres['indicators'][0]['observable'], 'address_value')
             except Exception:
-                data = self.get_recursively(eachres['observables']['observables'][0], 'value')
+                try:
+                    data = self.get_recursively(eachres['observables']['observables'][0], 'value')
+                except Exception:
+                    continue
```

Comment:
Why not throwing an error anymore?

Reply:
Because if nothing is found in this particular indicator, I still need the code to keep going. This is not actually an error but a faulty indicator and we can simply skip it, moving on to next indicator.

Reply:
Then let's add a debug log here.

Reply:
Done